python/hsfs/training_dataset.py

```diff
@@ -230,25 +229,11 @@ def _append_time_split(
                 TrainingDatasetSplit(
                     name=split_name,
                     split_type=TrainingDatasetSplit.TIME_SERIES_SPLIT,
-                    start_time=self._convert_event_time_to_timestamp(start_time),
-                    end_time=self._convert_event_time_to_timestamp(end_time),
+                    start_time=util.convert_event_time_to_timestamp(start_time),
+                    end_time=util.convert_event_time_to_timestamp(end_time),
                 )
             )
 
-    def _convert_event_time_to_timestamp(self, event_time):
-        if event_time is None:
-            return None
-        if isinstance(event_time, str):
-            if event_time:
-                return util.get_timestamp_from_date_string(event_time, timezone.utc)
-        elif isinstance(event_time, int):
-            if event_time == 0:
-                raise ValueError("Event time should be greater than 0.")
-            # jdbc supports timestamp precision up to second only.
-            return event_time * 1000
-        else:
-            raise ValueError("Given event time should be in `str` or `int` type")
-
     def save(
```

Comment:
`get_statistics` could have the conversion as well

Reply:
done